src/App.js

```diff
@@ -1,19 +1,112 @@
 import React from 'react';
 import './App.css';
-
+import todos from './api/todos';
 import users from './api/users';
+import { TodoList } from './components/TodoList/TodoList';
+
+const preparedTodos = todos.map(todo => ({
+  ...todo,
+  user: users.find(user => user.id === todo.userId),
+}));
+
+class App extends React.Component {
+  state={
+    todos: preparedTodos,
+    id: 3,
```

Comment:
if you change preparedTodos, you gonna need to change a number here, is better to use preparedTodos.length